unit_tests/src/test_file_object.cpp

```diff
@@ -2,9 +2,9 @@
 
 #include "irods/irods_file_object.hpp"
 
-TEST_CASE("file_object")
+TEST_CASE("file object")
 {
-    SECTION("test_file_object_size_can_be_negative__issue_7154")
+    SECTION("test file object size can be negative issue 7154")
```

Comment:
My bad. I was referring to everything except the type.

`file object` -> `file_object`